src/Nancy.Bootstrappers.StructureMap/StructureMapNancyBootstrapper.cs

```diff
@@ -1,3 +1,5 @@
+using Nancy.Configuration;

```

Comment:
It's not like it's something we recently introduced =)

And yes, we should get a, proper, `.dotsettings` file committed